nautobot/dcim/models/devices.py

```diff
@@ -955,6 +956,55 @@ def get_children(self):
         """
         return Device.objects.filter(parent_bay__device=self.pk)
 
+    @property
+    def modules(self):
```

Comment:
Maybe `all_modules` to make it clearer that this includes descendants recursively?